octavia-cli/octavia_cli/list/definitions.py

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+import abc
+from enum import Enum
+from typing import List, Union
+
+import airbyte_api_client
+from airbyte_api_client.api import destination_definition_api, source_definition_api
+
+
+class DefinitionType(Enum):
+    SOURCE = "source"
+    DESTINATION = "destination"
+
+
+class Definitions(abc.ABC):
```

Comment:
```suggestion
class ConnectorDefinitions(abc.ABC):
```
Maybe use a more specific naming compared to Definition? Just a suggestion

Reply:
I renamed it to `ConnectorsDefinitions`: d515b94de77351b977b14b19776118018d2790e8